code/src/Newest.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import moment from 'moment'
+
+export const Newest = ({ thoughts, onLikeClick }) => {
+  return (
+    <>
+      {thoughts.map((thought) => (
+        <div
+          className='message-container'
+          key={thought._id}
+          onClick={onLikeClick}
+        >
+          <p>{thought.message}</p>
+          <div className='icon-container'>
+            <div className='button-container'>
+              <button
+                className={
+                  thought.hearts > 0
+                    ? 'like-button like-button-clicked'
+                    : 'like-button'
+                }
```

Comment:
love the different classNames and styling depending on whether the message has gotten any likes